KarhooUISDK/Payments/PSPCore.swift

```diff
@@ -0,0 +1,15 @@
+//
+// Created by Bartlomiej Sopala on 06/05/2022.
+// Copyright (c) 2022 Flit Technologies Ltd. All rights reserved.
+//
+
+import Foundation
+import KarhooSDK
+
+public protocol PaymentManager {
+    var getCardFlow: CardRegistrationFlow { get }
```

Comment:
Names that start with `get` should be reserved for methods, not variables. A more adequate name would be `cardFlow`. Same for `getNonceProvider` > `nonceProvider`